packages/pkg-tests/yarn.test.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+const {
+  tests: {generatePkgDriver, startPackageServer, getPackageRegistry},
+  exec: {execFile},
+} = require(`pkg-tests-core`);
+
+const {basic: basicSpecs, dragon: dragonSpecs} = require(`pkg-tests-specs`);
+
+const pkgDriver = generatePkgDriver({
+  runDriver: (path, args, {registryUrl}) => {
+    const extraArgs = [`--cache-folder`, `${path}/.cache`];
+    return execFile(process.execPath, [`${process.cwd()}/../artifacts/yarn-1.4.1.js`, ...extraArgs, ...args], {
```

Comment:
Path is no longer valid since I moved `pkg-tests` into `packages/pkg-tests`, I need to update it

Reply:
The version changes constantly too. Too lazy to directly submit a PR to your branch... I wrote this to auto detect yarn:

```
// @flow

const fs = require('fs');
const {
  tests: {generatePkgDriver, startPackageServer, getPackageRegistry},
  exec: {execFile},
} = require(`pkg-tests-core`);

const {basic: basicSpecs, dragon: dragonSpecs} = require(`pkg-tests-specs`);

const yarnFileMatcher = /^yarn-\d+\.\d+\.\d+\.js$/;

const getYarnPath = function(): string {
  const artifactsFolder = `${process.cwd()}/../../artifacts`;
  const yarnFile = fs.readdirSync(artifactsFolder).find(file => yarnFileMatcher.test(file));
  if (!yarnFile) {
    throw new Error(`Unable to locate yarn in ${artifactsFolder}`);
  }
  return `${artifactsFolder}/${yarnFile}`;
};

const pkgDriver = generatePkgDriver({
  runDriver: (path, args, {registryUrl}) => {
    const extraArgs = [`--cache-folder`, `${path}/.cache`];
    const yarnPath = getYarnPath();
    return execFile(process.execPath, [yarnPath, ...extraArgs, ...args], {
      env: {[`NPM_CONFIG_REGISTRY`]: registryUrl, [`YARN_SILENT`]: `1`},
      cwd: path,
    });
  },
});

beforeEach(async () => {
  await startPackageServer();
  await getPackageRegistry();
});

basicSpecs(pkgDriver);
dragonSpecs(pkgDriver);
```